config/locales/in_person_proofing/fr.yml

```diff
@@ -145,7 +145,13 @@ fr:
         state_id_jurisdiction_hint: Il s’agit de l’État qui a émis votre pièce d’identité
         state_id_jurisdiction_prompt: '- Sélectionnez -'
         state_id_number: Numéro d’identification
-        state_id_number_hint: Peut comprendre des lettres et des chiffres
+        state_id_number_hint_html: "Il peut s’agir de lettres, de chiffres et des
+          symboles suivants: <span class='usa-sr-only'>des espaces, </span><span
+          aria-hidden='true'> </span> <span class='usa-sr-only'>des barres
+          obliques, </span><span aria-hidden='true'>/</span> <span
+          class='usa-sr-only'>des astérisques, </span><span
+          aria-hidden='true'>*</span> <span class='usa-sr-only'>des
+          tirets</span><span aria-hidden='true'>-</span>"
         zipcode: Code postal
```

Comment:
No more instances of `state_id_number_hint` found - all are `state_id_number_hint_html`

<img width="497" alt="Screenshot 2023-05-24 at 3 32 33 PM" src="https://github.com/18F/identity-idp/assets/125507397/30fe16a9-6992-4ab9-9607-b82cf1ae5da8">